Minor formatting (#1958)
SumoLogic/sumologic-documentation: docs/send-data/hosted-collectors/amazon-aws/amazon-cloudwatch-source-metrics.md

Tag filtering is only supported for user-defined AWS tags, not for AWS-generated tags.
:::

> Here’s how tag filtering works.

Reviewer: ```suggestion
Here’s how tag filtering works:
```

> * If you don't specify a tag filter for a namespace, the source will collect all metrics for the namespace.

Reviewer: ```suggestion
*Specify a tag filter for a namespace **OR** else the source will collect **all** metrics for the namespace.
```

Reviewer: Given the severity of the impact, make what they should do a command, and tell them why.
